crates/event-watcher-traits/src/evm/mod.rs

```diff
@@ -0,0 +1,45 @@
+// Copyright 2022 Webb Technologies Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#![warn(missing_docs)]
+//! # EVM Events Watcher Traits 🕸️
+
+use futures::prelude::*;
+use std::cmp;
+use std::ops::Deref;
+use std::sync::Arc;
+use std::time::Duration;
+
+use webb::evm::ethers::{
+    contract,
+    providers::{self, Middleware},
+    types,
+    types::transaction,
+};
+use webb_proposals::{ResourceId, TargetSystem, TypedChainId};
+use webb_relayer_context::RelayerContext;
+use webb_relayer_store::sled::SledQueueKey;
+use webb_relayer_store::{
+    BridgeCommand, BridgeKey, EventHashStore, HistoryStore, ProposalStore,
+    QueueStore,
+};
+use webb_relayer_utils::metric;
+
+/// Event watching traits
+pub mod event_watcher;
```

Comment:
make them private mods.
```suggestion
mod event_watcher;
```